### DISCUSSION TITLE
New feed take so much time to load.

### DISCUSSION THREAD

**Discussion by tejasness** [community/community#13884] [Other Feature Feedback, Questions, & Ideas]
after the new feed feature, my chrome hangs while loading the homepage of GitHub. it takes around 10 sec to load. previously before the implementation of the new feed, it was taking only a couple of seconds to load.

**Comment by tejasness** [comment]
Laptop: Toshiba Satellite C660
RAM: 3GB
Processor: Intel i3 first gen
Chrome version: 100

**Comment by jfullstackdev** [comment]
never had this on my Chrome, even just one moment,

anyways, 

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
